recipe/install_clang.bat

```diff
@@ -13,11 +13,22 @@ mkdir bin
 
 setlocal enabledelayedexpansion
 for /f "tokens=1 delims=." %%a in ("%PKG_VERSION%") do (
-  move bin2\clang.exe bin\clang-%%a.exe
+  set "MAJOR_VERSION=%%a"
 )
+move bin2\clang.exe bin\clang-!MAJOR_VERSION!.exe
 rmdir /s /q bin2
 
 cd %LIBRARY_BIN%
-for /f "tokens=1 delims=." %%a in ("%PKG_VERSION%") do (
-  copy clang-%%a.exe "clang++-%%a.exe"
+copy clang-!MAJOR_VERSION!.exe "clang++-!MAJOR_VERSION!.exe"
+
+FOR /F "tokens=* USEBACKQ" %%F IN (`%LIBRARY_PREFIX%\bin\clang.exe -print-resource-dir`) DO (
+   set "RESOURCE_DIR=%%F"
 )
```

Comment:
This fails currently
```
(C:\bld\clang_packages_1759496764226\_build_env) (base) C:\bld\clang_packages_1759496764226\_h_env\Library\bin>FOR /F "tokens=* USEBACKQ" %F IN (`C:\bld\clang_packages_1759496764226\_h_env\Library\bin\clang.exe -print-resource-dir`) DO (set "RESOURCE_DIR=%F" ) 
'C:\bld\clang_packages_1759496764226\_h_env\Library\bin\clang.exe' is not recognized as an internal or external command,
operable program or batch file.
```
which then causes `RESOURCE_DIR` to be unset and fail the comparison below:
```
"resource dir C:\bld\clang_packages_1759496764226\_h_env\Library\lib\clang\21 does not match expected "
```